onnxruntime/core/providers/openvino/ov_interface.cc

```diff
@@ -131,6 +131,13 @@ OVExeNetwork OVCore::StatefulCompileModel(std::shared_ptr<OVNetwork>& model,
     }
 
     UpdateNPUConfig(config, kv_pos, kv_desc);
+
+    if ((kv_desc.max_prompt_len + kv_desc.min_response_len) >= 2048) {
```

Comment:
Can you move this logic within UpdateNPUConfig(config, kv_pos, kv_desc) function call for config management & uniformity. Also instead of using a magic number like 2048 for comparison, can we introduce a compilation threshold like below -
```suggestion
    size_t npuw_context_len_threshold = 2048;
     if ((kv_desc.max_prompt_len + kv_desc.min_response_len) >= npuw_context_len_threshold) {
```

Reply:
By compilation threshold, you're saying you want it to be settable via some compilation flag?